gcp-secretmanager-pam.csproj

```diff
@@ -17,7 +17,13 @@
     <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="5.0.0" />
   </ItemGroup>
 
-  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+	<ItemGroup>
+		<None Update="manifest.json">
```

Comment:
i'll need to actually add the manifest.json for this copy to work, since this project did not previously have a manifest at all